adapters/syslog/syslog_test.go

```diff
@@ -41,8 +43,46 @@ var (
 	}
 	testTmplStr = fmt.Sprintf("<%s>%s %s %s[%s]: %s\n",
 		testPriority, testTimestamp, testHostname, testTag, testPid, testData)
+        hostHostnameFilename = "/etc/host_hostname"
+        hostHostnameContentWithoutLinefeed = "hostname"
+        hostHostnameContent = "hostname\r\n"
 )
 
+func TestHostnameDoesNotHaveLineFeed(t *testing.T) {
+	err := ioutil.WriteFile(hostHostnameFilename, []byte(hostHostnameContent), 0777)
+	if err != nil {
+		t.Fatal(err)
+	}
+	done := make(chan string)
+	addr, sock, srvWG := startServer("tcp", "", done)
```

Comment:
looks like that broke the reconnect test. No need to spin up the server. I'd just move the hostname logic to it's own function and test that specifically.

Reply:
Hey, I'm sorry, I really don't have time to do this. Maybe someone with more experience in go should do this.

Reply:
No worries. I'll take it from here. Thanks for the initial contribution!